static/app/views/alerts/rules/utils.tsx

```diff
@@ -113,3 +119,34 @@ export function shouldUseErrorsDiscoverDataset(
 
   return dataset === Dataset.ERRORS;
 }
+
+export function getExploreUrl({
+  rule,
+  orgSlug,
+  period,
+  projectId,
+}: {
+  orgSlug: string;
+  period: string;
+  projectId: string;
+  rule: MetricRule;
+}) {
```

Comment:
I think it would be good to make this a bit more generic. I'm looking at another area where we would want to link to explore, however, the `rule` prop sounds like an alert concept. Also, this util is under `rules` folder, but it could definitely be extracted out.

I think one of these two options is good
1. Move `getExploreUrl` to a more generic folder, update the params to be more generic.
2. Create a new `getExploreUrl` located in a generic utils folder, and use that function within the alerts version of `getExploreUrl`

I've done a url rename, and it was a pain finding all these different url sources, so it would be great to get this in sooner imo!

btw, This isn't a blocker, and i don't mind cleaning this up a bit as well! But lmk

Reply:
Beyond this PR, we could have one source of truth for the traces base url in some constant, this constant can be used within `routes.tsx`, `sidebar.tsx`, `getExploreUrl`, etc etc

Reply:
👍 good feedback, there's actually an existing similar helper function in dashboards already (`getWidgetExploreUrl.tsx`) so we can definitely consolidate this a bit better. I'm gunna follow up on this in a next pr!